x-pack/legacy/plugins/ml/public/components/chart_tooltip/chart_tooltip.tsx

```diff
@@ -12,7 +12,53 @@ import { TooltipValueFormatter } from '@elastic/charts';
 // eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { useObservable } from '../../../../../../../src/plugins/kibana_react/public/util/use_observable';
 
-import { chartTooltip$, mlChartTooltipService, ChartTooltipValue } from './chart_tooltip_service';
+import { chartTooltip$, ChartTooltipValue } from './chart_tooltip_service';
+
+type RefValue = HTMLElement | null;
+
+function useRefWithCallback() {
+  const chartTooltipState = useObservable(chartTooltip$);
+  const ref = useRef<RefValue>(null);
+
+  return (node: RefValue) => {
```

Comment:
should this be returning a named function called something like `setChartTooltipElement`?

Reply:
It's a wrapper for `useRef()` which just returns one object also, so I'd like to keep it the same (https://reactjs.org/docs/hooks-reference.html#useref)